src/lib/package.json

```diff
@@ -17,16 +17,16 @@
 	],
 	"license": "MIT",
 	"peerDependencies": {
-		"@angular/animations": "7.x",
+		"@angular/animations": "8.x",
 		"@angular/cdk": "8.x",
-		"@angular/common": "7.x",
-		"@angular/compiler": "7.x",
-		"@angular/core": "7.x",
-		"@angular/forms": "7.x",
+		"@angular/common": "8.x",
+		"@angular/compiler": "8.x",
+		"@angular/core": "8.x",
+		"@angular/forms": "8.x",
 		"@angular/material": "8.x",
-		"@angular/platform-browser": "7.x",
-		"@angular/platform-browser-dynamic": "7.x",
-		"@angular/router": "7.x",
+		"@angular/platform-browser": "8.x",
+		"@angular/platform-browser-dynamic": "8.x",
+		"@angular/router": "8.x",
 		"angular-l10n": "7.x",
```

Comment:
Was ist mit den anderen Libs?

Reply:
die sind auch in der app package.json nicht aktueller

Reply:
Das müsste noch passieren